app/assets/javascripts/discourse/tests/integration/components/highlighted-code-test.gjs

```diff
@@ -0,0 +1,42 @@
+import { render } from "@ember/test-helpers";
```

Comment:
test-helpers is such an odd home for this thing, maybe give it an official home under a discourse module.